.github/workflows/check-yaml-format.yaml

```diff
@@ -0,0 +1,28 @@
+name: Validate PR - YAML Format
```

Comment:
Isn't It would be best if the file started with ---

Reply:
Why?

Reply:
I think `---` makes file structure more explicit, clearer and readable, especially when dealing with complex yaml documents that contains multiple documents concatenated together. I don't think yamlfmt handles this. Correct me if I am wrong.